src/core/src/op/convert.cpp

```diff
@@ -33,7 +34,8 @@ struct Evaluate : public element::NoAction<bool> {
     template <element::Type_t ET_IN,
```

Comment:
Update comment above regarding supported types for this conversion.

Reply:
done